<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(18, 130, 119)</string>
  </property>
  <widget class="QPushButton" name="add_user_food">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Добавить свою еду</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_food">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Утка по пекински, 0, 0, 0, 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="app_logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poor Richard</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>MyDiet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Название, кол-во ккал, кол-во б, ж, у </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>(на 100 г продукта)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="food">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
